ycms/cms/views/zentral.py

```diff
@@ -0,0 +1,96 @@
+from django.contrib import messages
+from django.shortcuts import redirect, render
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ..constants import job_types
+from ..forms import WardForm
+from ..models import User, Ward
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+
+    template_name = "zentral.html"
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        This function returns a list of all wards,
+        as well as a form for creating a new one(not implemented yet)
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+
+        """
+        # ward list which acts as a iterable object
+        total_wards = Ward.objects.all()
+        # total number of wards
+        total_wards_count = Ward.objects.all().count()
+        # total number of beds
+        total_beds_count = total_wards_count * 28
+        # total occupied beds
+        total_occupied_beds = 0
+        total_available_beds = 0
+        for ward in total_wards:
+            total_occupied_beds += ward.occupied_beds
+            total_available_beds += ward.available_beds
```

Comment:
```suggestion
        total_occupied_beds = sum(ward.occupied_beds for ward in total_wards)
        total_available_beds = sum(ward.available_beds for ward in total_wards)
```
Your version works, of course, this is just a bit more concise / pythonic